rasa/cli/scaffold.py

```diff
@@ -47,6 +47,11 @@ def add_subparser(
         action="store_true",
         help="Temporary. Whether to create a DM2 project or a classic one",
     )
+    scaffold_parser.add_argument(
```

Comment:
rather than adding more different arguments, I think we should add one like `--template tutorial` or use a positional argument `rasa init tutorial`. I don't think adding a different parameter per variation makes sense.

Reply:
yes, but on the other hand once the dm2 version becomes the default we'll just go back to having either normal or tutorial project

Reply:
Nothing is more permanent than a provisorium. making it a property will also allow us in the future to e.g. use it with starter packs.